src/server/game/Entities/Object/Object.cpp

```diff
@@ -2697,6 +2697,13 @@ void WorldObject::GetCreatureListWithEntryInGrid(std::list<Creature*>& creatureL
     cell.Visit(pair, visitor, *(this->GetMap()), *this, maxSearchRange);
 }
 
+void WorldObject::GetPlayerListInGrid(std::list<Player*>& playerList, float maxSearchRange) const
+{
+    Trinity::AnyPlayerInObjectRangeCheck checker(this, maxSearchRange);
+    Trinity::PlayerListSearcher<Trinity::AnyPlayerInObjectRangeCheck> searcher(this, playerList, checker);
+    this->VisitNearbyWorldObject(maxSearchRange, searcher);
```

Comment:
@jackpoz No particular reason, it's just my code style. I believe we could use cell.Visit() here too, as is used in the other searcher functions.

@DDuarte My method will definitely replace that repeat code.